testing/local-amp-chrome-extension/popup.html

```diff
@@ -111,7 +111,7 @@
       box-sizing: border-box;
     }
 
-    label[for="base-url"], label[for="switch"] {
+    label[for="base-url"], #switch-label {
```

Comment:
to be consistent, can we switch both of these to a class (maybe ".section-header") ?

Reply:
Done in 6481452.